ServerCore/Helpers/RawHtmlHelper.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Microsoft.AspNetCore.Html;
+using Microsoft.AspNetCore.Mvc.Rendering;
+
+namespace ServerCore.Helpers
+{
+    public static class RawHtmlHelper
+    {
+        /// <summary>
+        /// Helper for extracting raw HTML from a string if present.
+        /// The syntax we look for is: {anything}Html.Raw({raw html})
+        /// </summary>
+        /// <param name="html">html to search for raw HTML in</param>
+        public static IHtmlContent GetRawHtmlIfPresent<T>(string html, IHtmlHelper<T> helper)
```

Comment:
The name seems a bit off to me. `IfPresent` suggests it doesn't do anything if html isn't present. Also as used it's `....Helper...RawHtmlHelper...RawHtml...` which is long and redundant. I'd suggest maybe something like `Helpers.Display.ForTextOrRawHtml` or `Helpers.Display.ForOptionalRawHtml` since the `Display...` functions return `IHtmlHelper` objects.

Reply:
The name made sense before I refactored all the IHtmlHelper logic into the method, and I forgot to change the method name. I agree it's redundant, I'm just going to call the method "Display".